estudios_socioeconomicos/utils.py

```diff
@@ -1,3 +1,15 @@
+import os
+
+
+def _delete_file(path):
+    """ Deletes file from filesystem.
+    Parameters:
+    -----------
+        path : A string that specify the location of the file.
+    """
+    if os.path.isfile(path):
```

Comment:
This will fail silently in case the path is not a file, maybe some feedback from the function should be added?, I understand that if the file is not there, it has already technically done it's job, so this is not a mandatory change